views/documentation.pug

```diff
@@ -0,0 +1,18 @@
+extends layout.pug
+
+block head
+	.navbar
```

Comment:
Both? Technically the nav it self should be a partial so u can include it on the index. And just have the navbar as default and include the nav in the default navbar